packages/graphile-build-pg/src/plugins/PgConnectionArgOrderByDefaultValue.js

```diff
@@ -36,8 +36,9 @@ export default (function PgConnectionArgOrderByDefaultValue(builder) {
       }
 
       const defaultValueEnum =
-        TableOrderByType.getValues().find(v => v.name === "PRIMARY_KEY_ASC") ||
-        TableOrderByType.getValues()[0];
+        TableOrderByType.getValues().find(
+          v => v.name === inflection.builtin("PRIMARY_KEY_ASC")
```

Comment:
Please extract this function call to just above the `defaultValueEnum` declaration

```js
const primaryKeyAsc = inflection.builtin("PRIMARY_KEY_ASC");
const defautlValueEnum = ...
```

and make this change in the other similar places too :+1:

Reply:
Done !